x-pack/legacy/plugins/siem/public/components/timeline/body/stateful_body.tsx

```diff
@@ -106,7 +104,6 @@ class StatefulBodyComponent extends React.PureComponent<StatefulBodyComponentPro
         addNoteToEvent={this.onAddNoteToEvent}
         browserFields={browserFields}
         id={id}
-        isLoading={isLoading}
         columnHeaders={columnHeaders || []}
```

Comment:
optional since I don't think we saw an impact while profiling, but perhaps we should replace `|| []` with a stable reference?